core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java

```diff
@@ -53,6 +55,11 @@ public EventType saveEventType(final EventTypeBase eventTypeBase) throws Interna
         }
     }
 
+    private String constructVersionString(final EventTypeBase eventTypeBase) {
```

Comment:
would be nice if schema know what type it is then we do not need that method. instead would just create it itself

Reply:
I have moved to type enum itself since that seemed like the most appropriate place

Reply:
Moved this to respective Version classes now